tests/cli-v2.rs

```diff
@@ -73,6 +74,39 @@ fn install_toolchain_from_version() {
     });
 }
 
+#[test]
+fn install_with_profile() {
+    clitools::setup(Scenario::Full, &|config| {
+        // Installing with minimal profile should only install rustc
```

Comment:
I wonder if we should add `rustup set profile complete` just to make it clear that the profile setting would install `cargo` too, but for the use of `--profile` in the subsequent command?